build.sbt

```diff
@@ -56,3 +54,61 @@ lazy val root = (project in file("."))
     resolvers += Resolver.typesafeIvyRepo("releases"),
     skip in publish := true
   )
+
+// publishing to maven central
+
+ThisBuild / organization := "com.soundcloud"
+ThisBuild / organizationName := "SoundCloud"
+ThisBuild / organizationHomepage := Some(url("https://developers.soundcloud.com/"))
+
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/soundcloud/twinagle"),
+    "scm:git@github.com:soundcloud/twinagle.git"
+  )
+)
+ThisBuild / developers := List(
+  Developer(
+    id    = "ccmtaylor",
+    name  = "Christopher Taylor",
+    email = "christopher.taylor@soundcloud.com",
+    url   = url("https://github.com/ccmtaylor")
+  ),
+  Developer(
+    id    = "oberkowitz",
+    name  = "Oren Berkowitz",
+    email = "elided@soundcloud.com",
+    url   = url("https://github.com/oberkowitz")
+  )
+)
+
+usePgpKeyHex("612C04F1EFE66FB7")
```

Comment:
this is the pgp key we'll use to sign releases. I guess it could also go in a separate file outside of git. OTOH, it's not secret and might be useful if somebody wants to make sure that artifacts in maven were signed by the right key?